adserver/models.py

```diff
@@ -2425,6 +2431,10 @@ class AdBase(TimeStampedModel, IndestructibleModel):
         null=True,
     )
 
+    # Whether this ad was rotated by the client.
+    # This is needed to measure CTR of rotated ads independently
+    rotated = models.BooleanField(_("Ad was rotated"), default=None, null=True)
```

Comment:
Do we want an int here to know how many rotations? Not sure we care, but might be interesting to compare at some point.

Reply:
Easy enough to switch